lib/matplotlib/projections/geo.py

```diff
@@ -268,6 +267,11 @@ def __init__(self, resolution):
             Transform.__init__(self)
             self._resolution = resolution
 
+        def __str__(self):
```

Comment:
Couldn't these be provided by a mixin class?  transform_path_non_affine also looks repeated across, haven't checked if other methods are too

Reply:
Possibly; I wasn't really looking for things to refactor, but I'm sure there's a lot of repetition in there.

I was also wondering if maybe we should `def __repr__(self): return str(self)` in some parent class instead of `__repr__ = __str__` in every subclass?

Reply:
That's a good point.  Want to push it?